azure-pipelines.yml

```diff
@@ -0,0 +1,92 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+jobs:
+- job: Linux
+
+  pool:
+    vmImage: 'Ubuntu 16.04'
+
+  strategy:
```

Comment:
For consistency across the different jobs (windows, linux, and mac), I would use "strategy" on each (maybe mac only has one matrix entry) and put all the common steps into a separate "-steps.yml" file